app/views/shared/_question.html.erb

```diff
@@ -1,3 +0,0 @@
-(line 1, column 17):
-unexpected ":"
-expecting white space or "=>"
```

Comment:
I think that was the output from haml1erb not understanding the original file ;)